src/hidimstat/utils.py

```diff
@@ -297,7 +297,7 @@ def _bhy_threshold(pvals, reshaping_function=None, fdr=0.1):
 
 
 ########################## alpha Max Calculation ##########################
-def _alpha_max(X, y, use_noise_estimate=False):
+def _alpha_max(X, y, use_noise_estimate=False, fill_diagonal=False, axis=None):
```

Comment:
When adding the argument axis, y is no longer required to be a column. We should refect that in the docstring for y.

Reply:
done